test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class NameTest {
+    
+    private Name name1;
+    private Name name2;
+    
+    @Test
+    public void testIsSimilar() {
+        // names are completely equal
+        name1 = new Name("Bob");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.